benchmarks-runner/build.gradle

```diff
@@ -0,0 +1,140 @@
+
+/*
+ * Copyright 2016-2019 JetBrains s.r.o.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+plugins {
+    id 'org.jetbrains.kotlin.jvm' version '1.3.40'
+}
+
+repositories {
+    mavenCentral()
+    jcenter()
+    maven { url 'https://dl.bintray.com/orangy/maven' }
+}
+
+dependencies {
+    implementation "org.jetbrains.kotlin:kotlin-stdlib"
+    implementation 'org.openjdk.jmh:jmh-core:1.21'
+    
+    runtimeOnly project(path: ':kotlinx-collections-immutable')
+}
+
+task copyBenchmarkList(type: Copy, dependsOn: ':benchmarks-mpp:jvmBenchmarkGenerate', group: "Benchmark") {
```

Comment:
Not holding the merge, but we need to investigate whether can we avoid copying this list and instead add that resources dir to the classpath.

Reply:
Ilya Ryzhenkov just helped me with this issue